### ISSUE TITLE
Cannot Add Missing from Unmapped Files

### ISSUE THREAD

**Issue by BGarber42** [Lidarr/Lidarr#4527]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Adding missing album info fails

### Expected Behavior

It works

### Steps To Reproduce

1) Add media to root folder
2) Wait for lidarr to fail to map it
3) Navigate to Unmapped Files
4) Click Add Missing

### Environment

```markdown
- OS: Unraid
- Lidarr: 2.1.4.3941
- Docker Install: Yes
- Using Reverse Proxy: No
- Database: Sqlite 3.41.2
```

### What branch are you running?

Develop

### Trace Logs?

https://sprunge.us/3QEt8w

### Trace Logs have been provided as applicable. Reports may be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by tcntad** [comment]
Same issue, it doesnt matter what metadata profile or whatever I click, nothing happens

**Comment by roks531** [comment]
How are there not more people complaining? I cant manually add any new folder that was added after the initial import :/. How is this not an essential feature.

Same problem on unmaped files, if you click add missing, nothing happens.

**Comment by bakerboy448** [comment]
- OP - Logs expired / not found
- tcntad - comments that add no value are spam.   Saying `me too` is spam.  For all we know you're running lidarr 0.6 or something as you stated no version number and added no useful information to this issue

**Comment by tcntad** [comment]
> * OP - Logs expired / not found
> 
>     * tcntad - comments that add no value are spam.   Saying `me too` is spam.  For all we know you're running lidarr 0.6 or something as you stated no version number and added no useful information to this issue

You're absolutely right, the issue is the same and I cant remember what version I had 7 months ago, today.
I reorganized my entire library and I believe what I cant map today is something Musicbrainz doesnt have
